### PR TITLE
Cson

### PR DESCRIPTION
**Pull request by dfroger**

### CODE REVIEW

File: recipes/cson/LICENSE [hunk 1]
```diff
@@ -0,0 +1,16 @@
+No-notice MIT License
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
```
Comment (bgruening) [inline recipes/cson/LICENSE]: This file is not needed, please remove it.

File: recipes/cson/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+test:
+  imports:
+    - cson
+
+about:
+  home: https://github.com/avakar/pycson
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (bgruening) [inline recipes/cson/meta.yaml]: `LICENSE` should be enough

File: recipes/speg/LICENSE [hunk 3]
```diff
@@ -0,0 +1,16 @@
+No-notice MIT License
+
```
Comment (bgruening) [inline recipes/speg/LICENSE]: Not needed here.

File: recipes/speg/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+test:
+  imports:
+    - speg
+
+about:
+  home: https://github.com/avakar/speg
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (bgruening) [inline recipes/speg/meta.yaml]: `LICENSE`

File: recipes/speg/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
+{% set name = "speg" %}
+{% set version = "0.2" %}
+{% set sha256 = "" %}
```
Comment (bgruening) [inline recipes/speg/meta.yaml]: checksum is missing here.

File: recipes/cson/meta.yaml (hunk 2 above)
Comment (dfroger) [inline recipes/cson/meta.yaml]: `LICENSE` file is not in the `cson-0.7.zip` file, so should I remove the file from the recipe directory and the `license_file` entry in the meta.yaml?

File: recipes/speg/meta.yaml (hunk 5 above)
Comment (dfroger) [inline recipes/speg/meta.yaml]: fixed!

File: recipes/cson/meta.yaml (hunk 2 above)
Comment (bgruening) [inline recipes/cson/meta.yaml]: Yes, please.

Comment (dfroger) [inline recipes/cson/meta.yaml]: done!

**Comment by dfroger** [comment]
ready to be reviewed again!

File: recipes/speg/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    - python
+
+test:
+  imports:
+    - speg
+
+about:
+  home: https://github.com/avakar/speg
+  license: MIT
+  license_family: MIT
```
Comment (jakirkham) [inline recipes/speg/meta.yaml]: Please ask upstream to add the license file to `MANIFEST.in`. Also please add the following here.

```yaml
  # Needs license file in the `sdist`.
  # Waiting on <issue/PR> to be addressed before uncommenting.
  # xref: <issue/PR link>
  # license_file: LICENSE

File: recipes/cson/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    - speg
+
+test:
+  imports:
+    - cson
+
+about:
+  home: https://github.com/avakar/pycson
+  license: MIT
+  license_family: MIT
```
Comment (jakirkham) [inline recipes/cson/meta.yaml]: Please ask upstream to add the license file to `MANIFEST.in`. Also please add the following here.

```yaml
  # Needs license file in the `sdist`.
  # Waiting on <issue/PR> to be addressed before uncommenting.
  # xref: <issue/PR link>
  # license_file: LICENSE

**Comment by jakirkham** [comment]
Need to ask upstream about getting license files in `MANIFEST.in` and link relevant issues and PRs back to here. Once that is addressed we can proceed.

File: recipes/cson/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - speg
```
Comment (ocefpaf) [inline recipes/cson/meta.yaml]: There will be a race condition b/c `speg` is added in this PR together with `cson`. You can just re-start the CIs later though.

**Pull request merged**